app/models/edition.rb

```diff
@@ -200,21 +197,21 @@ def self.latest_published_edition
   end
 
   def self.search_format_type
-    self.name.underscore.gsub('_', '-')
+    self.name.underscore.tr('_', '-')
```

Comment:
What's the reason for switching from [`gsub`](http://ruby-doc.org/core-2.2.3/String.html#method-i-gsub) to [`tr`](http://ruby-doc.org/core-2.2.3/String.html#method-i-tr)?